src/schema/__tests__/causality_jwt.test.js

```diff
@@ -1,51 +1,42 @@
 import jwt from "jwt-simple"
 import { omit } from "lodash"
-import schema from "schema"
 import { runQuery, runAuthenticatedQuery } from "test/utils"
 
 const { HMAC_SECRET } = process.env
 
 describe("CausalityJWT", () => {
-  const CausalityJWT = schema.__get__("CausalityJWT")
-
-  let gravity
+  let rootValue
 
   beforeEach(() => {
-    gravity = sinon.stub()
-    gravity.with = sinon.stub().returns(gravity)
-    gravity
-      .onCall(0)
-      .returns(Promise.resolve({ _id: "foo", name: "Foo sale", id: "slug" }))
-      .onCall(1)
-      .returns(
-        Promise.resolve({
-          _id: "craig",
-          paddle_number: "123",
-          type: "User",
-        })
-      )
-      .onCall(2)
-      .returns(
+    const me = {
+      _id: "craig",
+      paddle_number: "123",
+      type: "User",
+    }
+
+    const sale = { _id: "foo", name: "Foo sale", id: "slug" }
+
+    rootValue = {
+      saleLoader: sinon.stub().returns(Promise.resolve(sale)),
```

Comment:
Nit: these can just be plain functions

Reply:
Yea at some point I was realizing that when we're stubbing the entire `rootValue` that we can just do direct functions 👻